cmd/bosun/web/static/partials/config.html

```diff
@@ -123,7 +123,55 @@
 </ul>
 <div class="tab-content">
 	<div class="tab-pane" ng-class="{active: tab == 'timeline'}">
-		<div ts-alert-history ng-if="tab == 'timeline' && fromDate && toDate"></div>
```

Comment:
Did it make sense to extend ts-alert-history to know about templates instead of duplicating this?